docs/inputs.md

```diff
@@ -67,11 +63,12 @@ That said, any profile that uses the DSL keyword `input()` (or the deprecated `a
 
 ### How can I set Inputs?
 
-As installed (without specialized plugins), Chef InSpec supports five ways of setting inputs:
+As installed (without specialized plugins), Chef InSpec supports six ways of setting inputs:
```

Comment:
"several" ? @mjingle ?

Reply:
The change to "several" would be recommended. There's less maintenance this way.